jenkins/build.sh

```diff
@@ -149,15 +130,15 @@ yum erase -y systemd-python36-*
 
```

Comment:
You can also remove this line as we no longer install systemd-python36

Reply:
I have removed this from build.sh as we don't required this at the time of building rpm's.